ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -2,10 +2,30 @@
 import cv2
 import numpy as np
 
+import tensorflow as tf
+from helpers import load_graph
+
+import os.path as path
+
+# Load frozen graph of trained model
+five_up = path.abspath(path.join(__file__, '../../../../..'))
```

Comment:
I suggest the following option to load the model.
1. Store the model under tl_detector directory.
2. Dynamically fetch the path 
`import rospkg`
`base_path = r.get_path('tl_detector')`
`SSD_INCEPTION_SIM = path.join(base_path,'models/ssd_inception_v2_alex_sim/frozen_inference_graph.pb')`

Reply:
Done